activity_browser/ui/tables/models/parameters.py

```diff
@@ -283,6 +291,15 @@ def get_key(self, proxy: QModelIndex) -> tuple:
         index = self.proxy_to_source(proxy)
         return self._dataframe.iat[index.row(), self.key_col]
 
+    def remove_on_db_delete(self, name):
```

Comment:
Does this also delete the activity parameters specific to the database?

Reply:
No, this only deletes parameters on the 'activity' level when a database is deleted that contains activities with 'activity' level parameters. This didn't happen properly before, and 'activity' level parameters would remain without an activity or database. 

The parameters on database level are deleted on line 180 of this file.